Release 0 13 0 (#633)
kestra-io/docs: content/docs/05.developer-guide/02.tasks.md

|`timeout`| Task timeout expressed in [ISO 8601 Durations](https://en.wikipedia.org/wiki/ISO_8601#Durations). |
|`disabled`| Set it to `true` to disable execution of the task.                                                |
|`workerGroup.key`|To execute this task on a specific [Worker Group (EE)](../08.architecture.md#worker-group-ee).|

> |`logLevel`| To define the minimum flow log level for which a flow log will be inserted into the database. By default all logs are kept. |

Reviewer: ```suggestion
|`logLevel`| To define the log level granularity for which logs will be inserted into the backend database. By default, all logs are stored. However, if you restrict that to e.g., the `INFO` level, all lower log levels such as `DEBUG` and TRACE won't be persisted. |
```

> |`logLevel`| To define the minimum flow log level for which a flow log will be inserted into the database. By default all logs are kept. |

Reviewer: ```suggestion
|`logLevel`| To define the log level granularity for which logs will be inserted into the backend database. By default, all logs are stored. However, if you restrict that to e.g., the `INFO` level, all lower log levels such as `DEBUG` and TRACE won't be persisted. |
```

::

### ForEachItem

This task allows you to iterate over a list of items and run a subflow for each item, or for each batch containing multiple items.

Syntax:

```yaml
  - id: each
    type: io.kestra.core.tasks.flows.ForEachItem
    items: "{{ inputs.file }}" # could be also an output variable {{ outputs.extract.uri }}
    inputs:
      file: "{{ taskrun.items }}" # items of the batch
    batch:
      rows: 4
    namespace: dev
    flowId: subflow
    revision: 1 # optional (default: latest)
    wait: true # wait for the subflow execution
    transmitFailed: true # fail the task run if the subflow execution fails
    labels: # optional labels to pass to the subflow to be executed
      key: value
```

> This will execute the subflow `dev.subflow` for each batch of 10 items. 

Reviewer: ```suggestion
This will execute the subflow `dev.subflow` for each batch of items. 
```
4 vs 10 - no need to set this nr

> The subflow must define an input named `file` that will be set to the URI of the internal storage file containing the batch of items.

Reviewer: ```suggestion
To pass the batch of items to a subflow, you can use [inputs](../04.inputs.md). The example above uses an input of `FILE` type called `file` that takes the URI of an internal storage file containing the batch of items.
```

        console.log(colors.red("Hello"));
```

> This task can also fetch files from [namespace files][namespace files](./namespace-files.md) and make them available for all sub-tasks.

Reviewer: ```suggestion
This task can also fetch files from [namespace files][namespace files](./namespace-files.md) and make them available to all child tasks.
```

```yaml
id: node-with-cache
namespace: io.kestra.tests

tasks:
  - id: working-dir
    type: io.kestra.core.tasks.flows.WorkingDirectory
    namespaceFiles:

>       include: 

Reviewer: Do we need to add `enabled: true` or not? if not, then ✅ 
But it seems it's by default set to false so the filter wouldn't work without `enabled: true`
